### PR TITLE
Incorporate proofing changes into RMT Guide

### PR DESCRIPTION
**Pull request by lproven**
### PR creator: Description

Incorporate Dublin proofing corrections into the RMT guide. 

I have rejected one change ("newly-installed" → "newly installed") because I think it is correct as it stands, and the phrase is used in other places. I have also reflowed 2 chapters with `daps-xmlformat`.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 SP4/openSUSE Leap 15.4 *(if applicable to 15 SP4 _only_ -- do not merge yet!)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/rmt_mirroring.xml [hunk 1]
```diff
@@ -4,46 +4,43 @@
...
+  <para>
+   To change the default location of the mirrored repositories, point the
+   <filename>/usr/share/rmt/public/repo</filename> symbolic link to the desired
+   directory. This can be done using the <command>ln -sfn
+   <replaceable>TARGET</replaceable> /usr/share/rmt/public/repo </command>
+   command (replace <literal>TARGET</literal> with the desired destination).
+   Make sure that the target must have read and write permissions for the
+   <literal>rmt</literal> user and <literal>nginx</literal> group. If you
+   prefer to store mirrored repositories in a different directory, or on a
+   different storage device,
```
Comment (sknorr) [inline xml/rmt_mirroring.xml:35]: Comma at the end, the sentence is unfinished ... I guess you may want to fix that here too..?
Last changes:

```
e76e475b4b xml/rmt_mirroring.xml (Dmitri Popov       2020-05-20 13:07:27
```

File: xml/rmt_mirroring.xml [hunk 2]
```diff
@@ -88,17 +85,16 @@
...
+     To see the password, click the <inlinemediaobject>
+     <textobject role="description"><phrase>Eye icon</phrase>
+     </textobject>
+     <imageobject role="fo">
+      <imagedata fileref="scc_eye_icon.png" width="0.8em" format="SVG"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="scc_eye_icon.png" width="1em" format="PNG"/>
+     </imageobject>
+     </inlinemediaobject>icon.
```
Comment (sknorr) [inline xml/rmt_mirroring.xml:97]: A space after the icon:
```suggestion
     </inlinemediaobject> icon.
```

File: xml/rmt_mirroring.xml [hunk 3]
```diff
@@ -284,12 +287,12 @@ Repository by ID 3061 successfully disabled.
...
- <title>Enabling and disabling multiple repositories at once</title>
- <para>
-  To enable or disable multiple repositories at once, specify a space delimited
-  list of their IDs, for example:
- </para>
+   <tip>
+    <title>Enabling and disabling multiple repositories at once</title>
+    <para>
+     To enable or disable multiple repositories at once, specify a space
+     delimited list of their IDs, for example:
```
Comment (sknorr) [inline xml/rmt_mirroring.xml:294]:
```suggestion
     To enable or disable multiple repositories at once, specify a
     space-delimited list of their IDs, for example:
```

File: xml/rmt_rmt-cli.xml [hunk 4]
```diff
@@ -180,33 +179,33 @@
...
+      marked to be mirrored.
      </para>
     </listitem>
    </varlistentry>
   </variablelist>
  </sect2>
 
  <sect2 xml:id="sec-rmt-tools-rmt-cli-repos-custom">
-  <title><command>repos custom</command></title>
+  <title><command>repos&nbsp;custom</command></title>
```
Comment (sknorr) [inline xml/rmt_rmt-cli.xml:191]: This is a weird change. And if anyone copy-pastas this out of an HTML document into their terminal, they'll be unhappy with the result.

File: xml/rmt_rmt-cli.xml [hunk 5]
```diff
@@ -314,7 +313,7 @@
     <term><command>rmt-cli systems scc-sync</command></term>
     <listitem>
      <para>
-      This command forwards registered systems data to SCC.
+      This command forwards registered systems' data to SCC.
```
Comment (sknorr) [inline xml/rmt_rmt-cli.xml:316]: I think we'd usually do `systems's`...

File: xml/rmt_rmt-cli.xml [hunk 6]
```diff
@@ -362,17 +361,17 @@
...
   </para>
   <variablelist>
    <varlistentry>
     <term><command>rmt-cli export data [path]</command></term>
     <listitem>
      <para>
-      Run this on an online &rmt; to get the latest data from SUSE
-      Customer Center and save it as JSON files at the specified path.
+      Run this on an online &rmt; to get the latest data from SUSE Customer
+      Center and save it as JSON files at the specified path.
```
Comment (sknorr) [inline xml/rmt_rmt-cli.xml:374]: Use the `&scc;` entity here.

**Review by sknorr — COMMENTED** [review]
Random review nobody asked for

File: xml/rmt_mirroring.xml (hunk 1 above)
Comment (lproven) [reply xml/rmt_mirroring.xml:35]: I have looked at the 15.2 docs. It is the same there. The paragraph is not in the 15.1 docs. I do not know how to follow your reference. :-(

File: xml/rmt_rmt-cli.xml (hunk 4 above)
Comment (lproven) [reply xml/rmt_rmt-cli.xml:191]: The proofreader complained about a line break at this point in the TOC. I was attempting to fix it. It did not work; DAPS line-breaks it there anyway. 

I have reverted it.

File: xml/rmt_rmt-cli.xml (hunk 5 above)
Comment (lproven) [reply xml/rmt_rmt-cli.xml:316]: "Systems's" is **definitely** incorrect, and what I made was the recommendation from the proofreading team in Dublin, which is good, correct English.

Comment (sknorr) [reply xml/rmt_rmt-cli.xml:316]: You are right. The rule is that both `s'` and `s's` work for possessives of singulars (with Brits preferring `s'` and Americans split on the issue), and always `s'` when there's a plural possessive, such as this one.

File: xml/rmt_mirroring.xml (hunk 1 above)
Comment (sknorr) [reply xml/rmt_mirroring.xml:35]: Oh, my wording was a little bad. It was an extract from a `git blame` run  on the file.

Essentially, @dmpop was the person who added the unfinished sentence in May 2020 in commit e76e475.

Comment (dmpop) [reply xml/rmt_mirroring.xml:35]: This looks like a rogue sentence. Just remove _If you prefer to store mirrored repositories in a different
     directory, or on a different storage device,_ Thank you and sorry!

Comment (lproven) [reply xml/rmt_mirroring.xml:35]: Thanks! Fixed. I also put the same command on its own line and split off the run-on sentence.

**Review by cwickert — CHANGES_REQUESTED** [review]
Please have a look at my comments/suggestions.

Next time, please make content and format changes in separate commits. It's almost impossible to spot the proofing changes due to all the other changes.

File: xml/rmt_mirroring.xml (hunk 3 above)
Comment (lproven) [reply xml/rmt_mirroring.xml:296]: Fixed.

**Comment by lproven** [comment]
Thanks --

> Next time, please make content and format changes in separate commits. It's almost impossible to spot the proofing changes due to all the other changes.

Yes, @sknorr already made this point. My apologies for the extra work.

**Review by cwickert — APPROVED** [review]
Something went wrong when reordering the `list` and `list --all` sentences. In sec-rmt-mirroring-enable-disable-product a paragraph was added, in sec-rmt-mirroring-enable-disable-repository a sentence got lost. Not sure if I formatted my suggestions incorrectly or if was you fault. I just went ahead and fixed it.

**Comment by cwickert** [comment]
Oh, and there was a typo in the `rmt-cli products disable products` command, it read `product` instead. I fixed it, but unfortunately I made it worse be introducing it into the `enable` command, too. Fixed in 02fe041.

Since this bug exists in SLE 15 SP2 ([link](https://susedoc.github.io/doc-sle/SLE15SP2/html/SLES-rmt/cha-rmt-mirroring.html#sec-rmt-mirroring-enable-disable-product)) and possibly other branches, we should probably backport this PR accordingly.

**Pull request merged**